app/src/main/java/com/sdp/movemeet/utility/ActivityPictureCache.java

```diff
@@ -0,0 +1,86 @@
+package com.sdp.movemeet.utility;
+
+import android.Manifest;
+import android.app.Activity;
+import android.content.pm.PackageManager;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.graphics.Canvas;
+import android.graphics.Color;
+import android.graphics.drawable.BitmapDrawable;
+import android.graphics.drawable.Drawable;
+import android.net.Uri;
+import android.os.Build;
+import android.os.Environment;
+import android.util.Log;
+import android.view.View;
+import android.widget.ImageView;
+
+import androidx.core.app.ActivityCompat;
+
+import com.google.android.gms.tasks.Task;
+import com.google.firebase.storage.StorageTask;
+import com.sdp.movemeet.backend.firebase.storage.StorageImageManager;
+import com.sdp.movemeet.models.Image;
+import com.sdp.movemeet.view.activity.ActivityDescriptionActivity;
+import com.squareup.picasso.Picasso;
+
+import java.io.File;
+import java.io.FileOutputStream;
+
```

Comment:
You can add a comment in the beginning for explain the goal of this class

Reply:
Indeed !